sdk/include/psdk/shlguid.h

```diff
@@ -117,6 +117,8 @@ DEFINE_GUID(IID_IActiveDesktopP, 0x52502ee0, 0xec80, 0x11d0, 0x89, 0xab, 0x00, 0
 DEFINE_GUID(IID_IADesktopP2,     0xb22754e2, 0x4574, 0x11d1, 0x98, 0x88, 0x0, 0x60, 0x97, 0xde, 0xac, 0xf9);
 
 
+DEFINE_GUID(CLSID_FindFolder,
```

Comment:
This doesn't seem to exist in Microsoft's version of this header file, which means it doesn't belong in ours.
* If Windows uses the same CLSID but it's undocumented, this should go in shlguid_undoc.h (and should have a testcase in com_apitest)
* If Windows doesn't use this CLSID, it should likely go in some shell32-specific header, or it should possibly go away entirely.